### PR TITLE
[AP-5005] added docker file and nvmrc file

### PR DESCRIPTION
**Pull request by codingwithcn**
## Major Changes:
Only changes I made was:

```Docker
LABEL maintainer="DocNetwork <webteam@docnetwork.org>"
```
> Because ```MAINTAINER DocNetwork <webteam@docnetwork.org>``` was deprecated

## What To Test:
- Docker file, to make sure the file would work properly with a docker image, as intended.

### CODE REVIEW

**Comment by chriskemptondoc** [comment]
@mmclane , you're better at reading Dockerfiles than I.  Could you take a look at @codingwithcn 's work here, especially that use of LABEL?

File: .dockerfile
Comment (mmclane) [inline .dockerfile]: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

This label will work but all your doing is setting some metadata.  If you want to set it more formally you would use a label-schema.  

Ultimately this is an optional thing and doesn't really matter unless we are going to be sharing the container publicly.  You could remove it without any issue.

**Comment by chriskemptondoc** [comment]
One housekeeping item, @codingwithcn ... please prefix your commit messages and PR titles with the Jira ticket number.  I fixed this PR's title.

We also follow a convention of linking the Jira ticket in the description of the PR, along with a list of worthwhile tests for the reviewer to perform.

File: .dockerfile
Comment (mmclane) [inline .dockerfile]: Totally optional but, you could refactor this to be a little cleaner and a little more flexible.

On line 11 add ```ARG registry=dn-878788551012.d.codeartifact.us-east-1.amazonaws.com```

Then on lines 12, 13, and 14 you can replace that string with $registry

by doing this you set one place to update the registry but also you can pass in the value with the --build-arg parameter from the command line.  I will leave it to you to decide if you want to include the whole path or not.

**Review by mmclane — APPROVED** [review]

File: .dockerfile
Comment (codingwithcn) [reply .dockerfile]: I think using ```label-schema``` is a good idea since from what I know the repository is public and available to all.

Comment (codingwithcn) [reply .dockerfile]: Just tried it, its saying ```label-schema``` is an ```Unknown Instruction```

Comment (chriskemptondoc) [reply .dockerfile]: Hmm, what might cause that, @mmclane ?

Comment (mmclane) [reply .dockerfile]: That should be: 
LABEL org.opencontainers.image.authors="webteam@docnetwork.org"

Comment (chriskemptondoc) [reply .dockerfile]: @codingwithcn , Matt's suggestion matches Docker's documented current best practice for specifying the maintainer:

https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

Is this working for you?  If so, please commit the change so we can finish up this PR.

Comment (codingwithcn) [reply .dockerfile]: yeah it worked

**Review by chriskemptondoc — APPROVED** [review]
I see that @mmclane 's suggestions have been implemented.  Ready to merge, @codingwithcn !

**Pull request merged**
